h1st/model/fuzzy_logic/fuzzy_logic_rules.py

```diff
@@ -0,0 +1,100 @@
+import logging
+from typing import NoReturn
+
+import skfuzzy
+from skfuzzy import control as ctrl
+from skfuzzy.control.term import Term
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+class FuzzyLogicRules:
+    def __init__(self) -> None:
```

Comment:
Describe here what is a `FuzzyLogicRules`. Is it a collection of rules or one rule? Are variables within rules, or outside? Which is the better design? If you already have the concept of a collection of rules, shouldn’t you simply have `add()` instead of `add_rule()`? Unless you need to name it `add_rule()` because you’ve included extraneous capabilities into the class?

Reply:
FuzzyLogicRules has a property called rules (dictionary) and holds the collection of fuzzy rules.
FuzzyLogicRules also has a property called vars (dictionary) which holds the collection of fuzzy variables. 

In use case 1 where user don't have the 
In use case 2 where user have collection of rules already, then I think it make sense to have `add()`

Reply:
By "Describe here" I meant in the comments. Not here, to me.